charts/pega/templates/_pega-deployment.tpl

```diff
@@ -48,6 +48,9 @@ spec:
     metadata:
       labels:
         app: {{ .name }}
+        {{- if .root.Values.global.jdbc.podbinding }}
+        aadpodidbinding: {{ .root.Values.global.jdbc.podbinding }}
+        {{- end  }}
```

Comment:
Sumit, I don't think we need to expose the option for taking `addpodidbinding` or explicitly provide the label in deployment. The provision to add extra labels to the deployment is sufficient. See line # 54, labels can be added using `podLabels` option. Clients can use this configuration to add the required labels. We can update the documentation to use this option for adding more labels